front/src/applications/stdcm/utils/__tests__/findClosestOperationalPoint.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe, it, expect } from 'vitest';
+
+import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+
+import { findClosestOperationalPoint } from '../formatConflicts';
+
+const operationalPoints: SuggestedOP[] = [
+  {
+    opId: 'd9c92cb4-6667-11e3-89ff-01f464e0362d',
+    uic: 87747006,
+    ch: 'BV',
+    kp: '130+538',
+    ci: 747006,
+    trigram: 'GE',
+    offsetOnTrack: 301,
+    track: '60bca110-6667-11e3-81ff-01f464e0362d',
+    coordinates: [5.714214596139134, 45.191404467130226],
+    name: 'A',
+    positionOnPath: 100,
+  },
+  {
+    opId: 'd9b38600-6667-11e3-89ff-01f464e0362d',
+    uic: 87747006,
+    ch: 'FP',
+    kp: '130+100',
+    ci: 747006,
+    trigram: 'GE',
+    offsetOnTrack: 60,
+    track: '646dc21a-6667-11e3-81ff-01f464e0362d',
+    coordinates: [5.712418028581643, 45.195158714174944],
+    name: 'B',
+    positionOnPath: 200,
+  },
+  {
+    opId: 'd94a2af4-6667-11e3-89ff-01f464e0362d',
+    uic: 87747006,
+    ch: 'P2',
+    kp: '129+952',
+    ci: 747006,
+    trigram: 'GE',
+    offsetOnTrack: 170,
+    track: '60bc8cea-6667-11e3-81ff-01f464e0362d',
+    coordinates: [5.711846462951984, 45.19643525506182],
+    name: 'C',
+    positionOnPath: 300,
+  },
+];
```

Comment:
A lot of properties are optional, we can simplify the sample like that

```suggestion
const sampleOperationalPoint: SuggestedOP = {
  opId: 'id1',
  offsetOnTrack: 10,
  track: '123456',
  positionOnPath: 100,
};

const operationalPoints: SuggestedOP[] = [
  sampleOperationalPoint,
  {
    ...sampleOperationalPoint,
    positionOnPath: 200,
  },
  {
    ...sampleOperationalPoint,
    positionOnPath: 300,
  },
];
```